src/tests/services/project-manager.test.js

```diff
@@ -225,17 +238,21 @@ describe('service project manager', () => {
               _id
               projectManager {
                 _id
-                fullName
+                firstName
+                lastName
               }
             }
           }
         }
       `
+    }).expect((res) => {
+      expect(res.body).toHaveProperty(
+        'data.updateServiceById.record.projectManager'
+      )
+      expect(
+        res.body.data.updateServiceById.record.projectManager.fullName
```

Comment:
Update this test since you've dropped the `fullName` in favor of `firstName` and `lastName`